tests/x11/yast2_snapper.pm

```diff
@@ -0,0 +1,92 @@
+use base "x11test";
+use testapi;
+
+# Test for basic yast2-snapper functionality. It assumes the data of the
+# opensuse distri to be available at /home/$username/data (as granted by
+# console_setup.pm)
+
+# Helper for letting y2-snapper to create a snapper snapshot
+sub y2snapper_create_snapshot() {
+    my $self = shift;
+    my $name = shift || "Awesome Snapshot";
+   # Open the 'C'reate dialog and wait until it is there
+    send_key "alt-c";
+    assert_screen 'yast2_snapper-createsnapshotdialog', 100;
+    # Fill the form and finish by pressing the 'O'k-button
+    type_string $name;
+    sleep 2;
+    send_key "tab";
+    sleep 2;
+    send_key "tab";
+    sleep 2;
+    send_key "tab";
+    sleep 2;
+    type_string "a=1,b=2";
+    sleep 2;
+    send_key "alt-o";
+}
+sub run() {
+    my $self = shift;
+    # Make sure yast2-snapper is installed (if not: install it)
+    ensure_installed "yast2-snapper";
+    # Start an xterm as root
+    x11_start_program("xterm");
+    wait_idle;
+    become_root;
+    # Create a snapper config
+    script_run "btrfs subvolume create /tmp/testdata";
+    script_run "snapper create-config /tmp/testdata";
```

Comment:
Why are you creating a custom subvolume and snapper config? wouldn't it be better to test the defaults that should be created as part of the install when btrfs is the root filesystem? Otherwise bugs like bsc#922308 might slip by